src/receive_imf.rs

```diff
@@ -1804,9 +1805,25 @@ async fn lookup_chat_by_reply(
     };
     let parent_chat = Chat::load_from_db(context, parent_chat_id).await?;
 
+    // If chat is 1:1 but from_id is different to chat contact, then it's probably an alias reply
+    if to_ids.len() == 1 && parent_chat.typ == Chattype::Single {
+        let name = chat::get_other_chat_contacts(context, parent_chat_id)
+            .await?
+            .get(0)
+            .cloned();
+        if let Some(name) = name {
+            if from_id != name && name != ContactId::SELF {
+                mime_parser.set_override_sender_name(Some(name.get_stock_name(context).await));
```

Comment:
Not sure what to do here, should we just pretend that the non-alias sender is the sender of this message or should we actually show the alias address? Anyways, we should probably change from_id to the un-aliased contact here. Otherwise, the name is overridden again later anyways.